index.html

```diff
@@ -0,0 +1,550 @@
+
+          <p>By Cosima Mielke</p>
+
+          <p>
+            Artists and designers from across the globe once again got out
+            their favorite tools and created unique and inspiring wallpapers
+            to welcome the new month. The wallpapers are available with...
+          </p>
+        </div>
+
+        <div class="md">
+          <p>8 DAYS AGO</p>
+
+          <h3>
+            Using CSCS Scripting Language For Cross-Platform Development
+          </h3>
+
+          <p>By Vassili Kaplan</p>
+
+          <p>
+            In this article, Vassili Kaplan explains how to use a scripting
+            language to develop cross-platform mobile applications.
+          </p>
+        </div>
+
+        <div class="md">
+          <p>our friends and supporters</p>
+        </div>
+
+        <div class="lg">
+          <div class="lg-img"></div>
+
+          <h3>1,651 smashing members</h3>
+
+          <p>
+            A friendly community that keeps this website alive. With books,
+            webinars, discounts and networking — for just 1 coffee a month.
+            Join the family.
+          </p>
+
+          <button>Explore features →</button>
+
+          <p>Already have an account? Sign in!</p>
+        </div>
+
+        ​
+
+        <div class="md">
+          <p>9 DAYS AGO</p>
+
+          <h3>
+            How Smashing Magazine Manages Content: Migration From WordPress To
+            JAMstack
+          </h3>
+
+          <p>By Sarah Drasner</p>
+
+          <p>
+            WordPress adoption is massive. So why would a WordPress site
+            consider moving to JAMstack? In this technical case study, we’ll
+            cover what an actual WordPress migration looks like, using
+            Smashing Magazine itself! We’ll talk through the gains and ….
+          </p>
+        </div>
+
+        <div class="md">
+          <p>9 DAYS AGO</p>
+
+          <h3>
+            Smashing Podcast Episode 7 With Stephanie Stimac And Aaron
+            Gustafson: What’s New In Microsoft Edge?
+          </h3>
+
+          <p>By Drew McLellan</p>
+
+          <p>
+            We’re taking a look at the brand new version of Microsoft Edge and
+            its Chromium-based rendering engine. What does that change mean
+            for web developers, and how does the death of Microsoft’s own
+            rendering engine impact the web? Drew McLellan talks to …
+          </p>
+        </div>
+
+        <div class="md">
+          <p>10 DAYS AGO</p>
+
+          <h3>How To Design Powerful Narratives On Mobile</h3>
+
+          <p>By Suzanne Scacca</p>
+
+          <p>
+            Want to build a website or PWA that attracts a large audience and
+            then converts as much of that audience into paying customers as
+            possible? If that’s the case, what you need to do is use
+            storytelling in your design.
+          </p>
+        </div>
+
+        <div class="md">
```

Comment:
line #213 should be `<article>` and not a `<div>`

Reply:
done